### ISSUE TITLE
unfortunately epub2go.eu has not been working for some time now

### ISSUE THREAD

**Issue by gutenbergbookworm** [nemiah/epub2go#1]
unfortunately epub2go.eu has not been working for some time now

**Comment by nemiah** [comment]
Yes, the problem is known and the current maintainer is working on it. It'll take a while I think

**Comment by daisuwio33-ui** [comment]
epub2go does not work any longer with the new projekt-gutenberg.org website

**Comment by gutenbergbookworm** [comment]
In the meantime, several EPUB-formats are available on projekt-gutenberg.org website.
